cwms_radar_api/src/main/java/cwms/radar/data/dto/Pool.java

```diff
@@ -0,0 +1,123 @@
+package cwms.radar.data.dto;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import usace.cwms.db.dao.ifc.pool.PoolNameType;
+import usace.cwms.db.dao.ifc.pool.PoolType;
+
+public class Pool extends PoolType implements CwmsDTO
+{
+	private Number attribute;
+	private String description;
+
+	private String clobText;
+
+	public Pool(PoolNameType poolName, String projectId, String bottomLevelId, String topLevelId, boolean implicit)
+	{
+		super(poolName, projectId, bottomLevelId, topLevelId, implicit);
+	}
+
+	public Pool(PoolType p)
+	{
+		super(p.getPoolName(), p.getProjectId(), p.getBottomLevelId(), p.getTopLevelId(), p.isImplicit());
+	}
+
+	public Number getAttribute()
+	{
+		return attribute;
+	}
+
+	public void setAttribute(Number attribute)
```

Comment:
The DTOs should be read only, use the builder pattern if everything can't be handled in the constructor.

Reply:
ok. I reworked Pool and Pools